Packetery/Checkout/Model/ResourceModel/Order/Collection.php

```diff
@@ -16,4 +16,30 @@ protected function _construct()
     {
         $this->_init('Packetery\Checkout\Model\Order', 'Packetery\Checkout\Model\ResourceModel\Order');
     }
+
+    /**
+     * @return string
+     */
+    private function createSalesOrderCarrierCodeCondition(): string {
+        $carrierCodes = \Packetery\Checkout\Model\Carrier\Facade::getAllCarrierCodes();
+
+        $carrierCodesConditions = [];
+        foreach ($carrierCodes as $carrierCode) {
+            $carrierCodesConditions[] = "sales_order.shipping_method LIKE " . $this->getConnection()->quote($carrierCode . '\_%', 'STRING');
```

Comment:
Pokud tomu nebrání technické omezení, prosím použít backticks na všechny názvy sloupců.

Reply:
Tady to být může. V SearchResult už ne. Tam to už magento dává